lib/routes/app-sales/util.ts

```diff
@@ -0,0 +1,160 @@
+import { type DataItem } from '@/types';
+
+import { art } from '@/utils/render';
+import ofetch from '@/utils/ofetch';
+
+import { type CheerioAPI, type Cheerio, type Element, load } from 'cheerio';
+import path from 'node:path';
+
+const baseUrl: string = 'https://app-sales.net';
```

Comment:
```suggestion
const baseUrl: string = 'https://www.app-sales.net';
```
Use the [redirected URL](https://github.com/DIYgod/RSSHub/actions/runs/14977198399/job/42072406160#step:13:19) instead.